packages/investment-pool/contracts/mock/FakeGovernancePoolMock.sol

```diff
@@ -0,0 +1,59 @@
+// @ buidl.one 2022
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.9;
+
+import {IGovernancePool} from "../interfaces/IGovernancePool.sol";
+import {IInvestmentPool} from "../interfaces/IInvestmentPool.sol";
+
+contract FakeGovernancePoolMock is IGovernancePool {
```

Comment:
Fake and mock in the name kinda serve the same purpose, no?
Perhaps we should leave only one option